client/assets/stylesheets/_wp-components-overrides.scss

```diff
@@ -22,6 +22,10 @@
 	background-color: var(--color-accent);
 	border-color: var(--color-accent);
 	color: var(--color-text-inverted);
+	border-radius: 4px;
+	font-size: 0.875rem;
+	font-weight: 500;
+	justify-content: center;
```

Comment:
I think this will apply to every `wordpress/components` `Button` in calypso?

We should be coordinating this kind of change with matching updates to `automattic/components` which is [using a 2px border-radius](https://github.com/Automattic/wp-calypso/blob/trunk/packages/components/src/button/style.scss#L23).

Maybe we split this off into a new PR?